modules/rosa-configure.adoc

```diff
@@ -0,0 +1,279 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/rosa_cli/get-started-with-rosa.adoc
+
+[id="rosa-configure_{context}"]
+= Configuring `rosa`
+
+Use the following commands to configure the `rosa` CLI (`rosa`).
+
+[id="rosa-login_{context}"]
+== login
+
+Logs in to your Red Hat account, saving the credentials to the `rosa` configuration file. You must provide a token when logging in. You can copy your token by from link:https://cloud.redhat.com/openshift/token/moa[the {product-title} token page].
+
+`rosa` looks for a token in the following priority order:
+
+* command line arguments
+* The `ROSA_TOKEN` environment variable
+* The `rosa` configuration file
+* Interactively from a command line prompt
+
+.Syntax
+[source,terminal]
+----
+$ rosa login [arguments]
+----
+
+.Arguments
+|===
+|Option |Definition
+
+|--client-id
+|The OpenID client identifier(string). Default: cloud-services
+
+|--client-secret
+|The OpenID client secret (string).
+
+|--env
+|The environment of the API gateway. The value is a string that can be the complete URL or an alias. Valid aliases: production, staging, integration. Default: https://api.openshift.com
+
+|--insecure
+|Enables insecure communication with the server. This disables verification of TLS certificates and host names.
+
+|--scope
+|The OpenID scope (string). If this option is used, it completely replaces the default scopes. This can be repeated multiple times to specify multiple scopes. Default: openid
+
+|-t, --token
+|Accesses or refreshes the token (string).
+
+|--token-url
+|The OpenID token URL (string). Default: https://sso.redhat.com/auth/realms/redhat-external/protocol/openid-connect/token
+|===
+
+.Optional arguments inherited from parent commands
+|===
+|Option |Definition
+
+|-h, --help
+|Shows help for this command.
+
+|debug
+|Enables debug mode.
+
+|--profile
+|Uses a specific AWS profile (string) from your credentials file.
+
+|-v, --v level
+|Log level for V logs.
+|===
+
+.Examples
+Login to the OpenShift staging API with an existing token.
+
+[source,terminal]
+----
+$ rosa login --env staging --token=$OFFLINE_ACCESS_TOKEN
+----
+
+Switch environments using an account where you are already logged in.
+[source,terminal]
+----
+$ rosa login --env production
+----
+
+[id="rosa-logout_{context}"]
+== logout
+
+Logs out of `rosa`. Logging out also removes the `rosa` configuration file.
```

Comment:
This is describing what the command does, so this verb tense is consistent throughout the doc. I believe this is accurate because I checked the IBM Style Guide examples for command line descriptions. LMK if I'm wrong.

Reply:
I see now how you did it and that you do have the tense consistent in the description that follows the command. So yes you are right :)